### PR TITLE
Add Wechat configuration

### LINKED ISSUE #1005: Document WeChat notification support
WeChat support got added in https://github.com/prometheus/alertmanager/pull/1059, but it's not documented yet on https://prometheus.io/docs/alerting/configuration/

### PR DESCRIPTION
**Pull request by simonpasquier**
Fixes #1005.

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
 this feature.
+
+## `<wechat_config>`
+
+Wechat notifications are sent via the [Wechat
+API](http://admin.wechat.com/wiki/index.php?title=Customer_Service_Messages).
+
+```yaml
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: Why are we sending resolved messages to a chat system?

File: content/docs/alerting/configuration.md [hunk 2]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
+
+# The Wechat API URL.
+[ api_url: <string> | default = global.wechat_api_url ]
+
+# The corp id for authentication
+[ corp_id: <string> | default = global.wechat_api_corp_id ]
+
+# API request data as defined by the Wechat API.
+[ message: <string> | default = '{{ template "wechat.default.message" . }}' ]
+[ agent_id: <string> | default = '{{ template "wechat.default.agent_id" . }}' ]
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: This doesn't seem right, this field isn't templated so why does it have that default?

Comment (simonpasquier) [reply content/docs/alerting/configuration.md]: Yep, you're right.

Ping @stuartnelson3 @berlinsaint @songjiayang

Comment (songjiayang) [reply content/docs/alerting/configuration.md]: Hi, in work WeChat,  a `agent_id ` means a chat room,  so I think `agent_id` should be a categoric value,  no need template, right ?

Comment (simonpasquier) [reply content/docs/alerting/configuration.md]: Then I've got https://github.com/prometheus/alertmanager/pull/1253 to remove those.

File: content/docs/alerting/configuration.md (hunk 1 above)
Comment (brian-brazil) [reply content/docs/alerting/configuration.md]: This should be changed too.

**Comment by songjiayang** [comment]
LGTM

File: content/docs/alerting/configuration.md [hunk 3]
```diff
@@ -507,3 +512,29 @@ endpoint:
 There is a list of
 [integrations](/docs/operating/integrations/#alertmanager-webhook-receiver) with
 this feature.
+
+## `<wechat_config>`
+
+Wechat notifications are sent via the [Wechat
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: "Wechat" -> "WeChat"

File: content/docs/alerting/configuration.md [hunk 4]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
+API](http://admin.wechat.com/wiki/index.php?title=Customer_Service_Messages).
+
+```yaml
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = false ]
+
+# The API key to use when talking to the Wechat API.
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: "Wechat" -> "WeChat"

File: content/docs/alerting/configuration.md [hunk 5]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
+[ api_secret: <secret> | default = global.wechat_secret_url ]
+
+# The Wechat API URL.
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: "Wechat" -> "WeChat"

File: content/docs/alerting/configuration.md [hunk 6]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
+[ api_url: <string> | default = global.wechat_api_url ]
+
+# The corp id for authentication
+[ corp_id: <string> | default = global.wechat_api_corp_id ]
+
+# API request data as defined by the Wechat API.
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: "Wechat" -> "WeChat"

File: content/docs/alerting/configuration.md [hunk 7]
```diff
@@ -507,3 +512,29 @@ endpoint:
...
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = false ]
+
+# The API key to use when talking to the Wechat API.
+[ api_secret: <secret> | default = global.wechat_secret_url ]
+
+# The Wechat API URL.
+[ api_url: <string> | default = global.wechat_api_url ]
+
+# The corp id for authentication
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: Period at end of comment line.

**Comment by simonpasquier** [comment]
@juliusv thanks for the review!

**Comment by stuartnelson3** [comment]
I'll wait for @juliusv to approve/merge

**Pull request merged**